app/libs/smarty/plugins/modifier.sanitizeRichtext.php

```diff
@@ -24,7 +24,9 @@
  * @return string
  */
 
- require_once getenv('APP_LIBS_PATH') . '/loaders/Leaf_autoloader.php';
+ if(!class_exists('App\Leaf\XSSHelpers')){
+    include_once __DIR__ . '/../../../Leaf/XSSHelpers.php';
```

Comment:
Looks like we have a few different iterations of this, qrcode in nexus vs portal are different to eachother and what is here for loading xsshelpers.

Reply:
Do we want a separate PR for the Nexus?

Reply:
I fixed the qrcode differences. Here the only thing needed is the XSSHelper so that is all we are loading. In the other cases we are loading globals and others because they are all needed.

@raphaelfretter I was only told to separate moved/copied files from modified files.